docs/source/configuration.md

```diff
@@ -81,22 +88,58 @@ Following fields are required (and can be customized) in the [`challenge_config.
+  - **test_annotation_file**: This file is used for ranking the submission made by a participant. An annotation file can be shared by more than one challenge phase. (Path of the test annotation file relative to this YAML file, e.g. `annotations/test_annotations_devsplit.json`)
 
   - **codename**: Unique id for each challenge phase. Note that the codename of a challenge phase is used to map the results returned by the evaluation script to a particular challenge phase. The codename specified here should match with the codename specified in the evaluation script to perfect mapping.
 
-  - **max_submissions_per_day**: Positive integer which tells the maximum number of submissions per day to a challenge phase.
-
-  - **max_submissions_per_month**: Positive integer which tells the maximum number of submissions per month to a challenge phase.
-
-  - **max_submissions**: a Positive integer that decides the overall maximum number of submissions that can be done to a challenge phase.
+  - **max_submissions_per_day**: Positive integer which tells the maximum number of submissions per day to a challenge phase.(Optional Default value is `100000`)
+
+  - **max_submissions_per_month**: Positive integer which tells the maximum number of submissions per month to a challenge phase.(Optional Default value is `100000`)
+
+  - **max_submissions**: A positive integer that decides the maximum number of total submissions that can be made to the challenge phase.(Optional Default value is `100000`)
+
+  - **default_submission_meta_attributes**: These are the default metadata attributes that will be displayed for all submissions, the metadata attributes are method_name, method_description, project_url, and publication_url.
+      ```yaml
+    default_submission_meta_attributes:
+      - name: method_name
+        is_visible: True
+      - name: method_description
+        is_visible: True
+      - name: project_url
+        is_visible: True
+      - name: publication_url
+        is_visible: True
+      ```
+  - **submission_meta_attributes**: These are the custom metadata attributes that can be added by participants for their submissions. The custom metadata attributes are TextAttribute, SingleOptionAttribute, MultipleChoiceAttribute, and TrueFalseField.
+    ```yaml
+    submission_meta_attributes:
+      - name: TextAttribute
+        description: Sample
+        type: text
+        required: False
+      - name: SingleOptionAttribute
+        description: Sample
+        type: radio
+        options: ["A", "B", "C"]
+      - name: MultipleChoiceAttribute
+        description: Sample
+        type: checkbox
+        options: ["alpha", "beta", "gamma"]
+      - name: TrueFalseField
+        description: Sample
+        type: boolean
+        required: True
+    ```
+  - **is_restricted_to_select_one_submission**: True/False (indicates whether participants are restricted to submitting only one submission for the challenge. Default is `False`)
```

Comment:
The description should be 'indicates whether to restrict user to select only one submission for leaderboard'.